crates/orchestrator/src/workers/update_state.rs

```diff
@@ -94,14 +94,18 @@ impl Worker for UpdateStateWorker {
                 }
             }
             None => {
-                if blocks_to_process[0] != 0 {
+                if blocks_to_process[0] != 0 && blocks_to_process[0] != 1 {
```

Comment:
let's not push this for now? or we are doing it maybe we should add a todo?

Reply:
I agree, this is temporary because snos is not able to create proof for block 0 and hence either the mongoDB has a block 0 state transition completed or we have this

Reply:
sure, removed.